src/app/authentication/interceptor/auth.interceptor.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { Observable } from "rxjs";
+import { IUserEntyty } from "src/app/interfaces/user-entyty.interface";
+import { AuthServiceService } from "../services/auth-service.service";
+
+@Injectable()
+export class AuthInterceptor implements HttpInterceptor {
+
+    userData: IUserEntyty;
+    constructor(private authService: AuthServiceService) { }
+
+    intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+
+        async () => {
+            let response = await this.authService.getUserInfo('taras@.com');
```

Comment:
interceptor doesn't work with fetch but... wtf)
I just skip this file now due to future rewriting